src/videojs-fairplay.js

```diff
@@ -212,12 +237,38 @@ class Html5Fairplay {
       status,
     } = event.target;
 
+    const {
+      licenseResponseHeaders,
+      licenseResponseErrorContent,
+    } = this.protection_;
+
     if (status !== 200) {
       this.onRequestError(event.target, ERROR_TYPE.FETCH_LICENCE);
 
+      // Return response content
+      if (licenseResponseErrorContent && typeof licenseResponseErrorContent === 'function') {
+        licenseResponseErrorContent(this.ab2str(event.target.response));
+      }
+
       return;
     }
 
+    // Return response headers here
+    if (licenseResponseHeaders && typeof licenseResponseHeaders === 'function') {
+      const headers = event.target.getAllResponseHeaders().split('\r\n').reduce((all, part) => {
```

Comment:
i would chuck this in a method somewhere to facilitate legibility and if its needed in another method.

`createHeadersFromResponseHeaders` or something.